lib/crawl.js

```diff
@@ -81,12 +92,16 @@ function crawl(url, flags) {
     // the correct mimetypes/headers
     crawler.on('fetchcomplete', queueItem => {
         if (queueItem.stateData.contentType.indexOf('text/html') > -1) {
-            urls.push({
-                label: queueItem.path,
-                url: queueItem.url,
-                selectors: ['document'],
-                hideSelectors: ['iframe'],
-            });
+            // Clone the default scenario and adjust urls as needed
+            const currentScenario = JSON.parse(JSON.stringify(defaultScenario));
```

Comment:
I think a more idiomatic approach would be `const currentScenario = Object.assign({}, defaultScenario);` But this is fine for now

Reply:
> I think a more idiomatic approach would be const currentScenario = Object.assign({}, defaultScenario);

This has been updated. Let me know if there is anything else I can do. Would love to get this merged in and have a new version tagged so I can start using `backstop-crawl` in my project!